ts/updater/updater.ts

```diff
@@ -46,7 +46,13 @@ export async function start(
   }, 1000 * 60 * 10); // trigger and try to update every 10 minutes to let the file gets downloaded if we are updating
   stopped = false;
 
-  await checkForUpdates(getMainWindow, messages, logger);
+  global.setTimeout(async () => {
```

Comment:
Doesn't this only need to run once? Can you explain why we have two intervals now?

Reply:
we tick every 10 minutes, and the first tick 2 minutes after start. there is only one interval

Reply:
we tick every 10 minutes, and the first tick 2 minutes after start. there is only one interval

Reply:
Cool. LGTM